explorer/app/config/model.ts

```diff
@@ -25,6 +26,16 @@ export interface EntityConfig<
   staticLoad?: boolean;
 }
 
+export interface ComponentObject<
```

Comment:
Can you add TSDocs here to help explain a little about the interface itself and also how the types work?